docker/injector/samples/14_airflow_ingestion_aqa.json

```diff
@@ -2,8 +2,8 @@
   "data_source_oddrn": "//airflow/host/aws.airflow.aws.com/air",
   "items": [
     {
-      "oddrn": "//airflow/host/aws.airflow.aws.com/flights/_!+AirFlights",
-      "name": "_!+AirFlights",
+      "oddrn": "//airflow/host/aws.airflow.aws.com/flights/_AirFlights",
+      "name": "_AirFlights",
```

Comment:
I would propose leaving at least one of the special symbols ("!" or "+") in this name. 

Because "_AirFlights" -- string which could be considered (from the ODD platform viewpoint) as a string without a special symbol